saleor/webhook/payloads.py

```diff
@@ -406,11 +406,16 @@ def generate_product_variant_payload(product_variants: Iterable["ProductVariant"
             "channel_listings": lambda v: json.loads(
                 generate_product_variant_listings_payload(v.channel_listings.all())
             ),
+            "stocks": lambda v: generate_product_variant_stocks_payload(v),
```

Comment:
I am not sure of this.  You return the quantity that is assigned to the stock in `stocks` field.  If we will want to extend variant's payload in the future by adding stocks data, we won't be able to extend this field as it stores int value currently. IMO it should be a custom field like `quantity_in_stocks` or something